object/search.go

```diff
@@ -271,6 +291,8 @@ func (f *SearchFilters) addPhyFilter() {
 	f.addFlagFilter(fKeyPropPhy)
 }
 
+// AddPhyFilter adds filter to object types.
```

Comment:
what types? can be confused with TS, SG, and REGULAR types IMO

Reply:
Updated